Update/omot_014.txt

```diff
@@ -2066,7 +2066,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500723", 256, TRUE);
 	OutputLine(NULL, "「仕方がないのです。工事現場の監督は、現場の人にも評判が悪かったので口論から発展、と想像できました。",
-			NULL, "\"It can't be helped. It's easy to imagine the fight with the on-site construction manager developed due to bad relations with the people at the construction site.", GetGlobalFlag(GLinemodeSp));
+			NULL, "\"It can't be helped. It's easy to imagine the fight with the on-site construction manager developed due to a argument based on his bad relations with the people at the construction site.", GetGlobalFlag(GLinemodeSp));
```

Comment:
```suggestion
			NULL, "\"It can't be helped. It's easy to imagine the fight with the on-site construction manager developed due to an argument based on his bad relations with the people at the construction site.", GetGlobalFlag(GLinemodeSp));
```